q2_metadata/templates/tabulate/index.html

```diff
@@ -3,22 +3,59 @@
 {% block content %}
   <div class="row">
     <div class="col-lg-12">
-      <p>Export options:</p>
-      <table id='table' class='table table-hover'></table>
+      <p>
+        <a href="metadata.tsv" target="_blank" rel="noopener noreferrer" class="btn btn-default">
+          Download normalized (and if applicable, merged) metadata
+        </a>
+        <br>
+        This file won't necessarily reflect dynamic sorting options based on
```

Comment:
Can you add a little more padding between the button and this text?

Can you also mention filtering in the text, e.g. "This file won't necessarily reflect dynamic sorting or filtering options based on the interactive table below."

Reply:
padding:
![screen shot 2018-02-14 at 10 31 01 am](https://user-images.githubusercontent.com/274668/36218604-381b01c0-1172-11e8-8fd7-9d799648dc32.png)